tests/tracer/test_ext.py

```diff
@@ -21,7 +25,153 @@ def _updateenv(monkeypatch, env):
         monkeypatch.setenv(str(k), str(v))
 
 
+@pytest.fixture
+def git_repo(tmpdir):
+    """Create temporary git directory, with one added file commit with a unique author and committer."""
+    cwd = str(tmpdir)
+    subprocess.check_output("git init", cwd=cwd, shell=True)
+    subprocess.check_output('git remote add origin "git@github.com:test-repo-url.git"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.name "Jane Doe"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.email "jane@doe.com"', cwd=cwd, shell=True)
+    subprocess.check_output("touch tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output("git add tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output(
```

Comment:
a comment would be nice, or maybe make the `Jane Doe` vs `John Doe` much more obvious.

Reply:
`Author McAuthorface` vs `Comitter McComitterface` :trollface: ?